jupyterhub/templates/proxy/service.yaml

```
@@ -63,6 +63,9 @@ spec:
       {{- with .Values.proxy.service.nodePorts.http }}
       nodePort: {{ . }}
       {{- end }}
+    {{- with .Values.proxy.extraPublicPorts }}
+    {{- . | toYaml | trimSuffix "\n" | nindent 4 }}
+    {{- end}}
```

I think we should nest this under `proxy.service`, which in values.yaml also should get an empty array as a default value, along with an entry in lint-and-validate-values.yaml where we put some mocked port value.

```suggestion
    {{- with .Values.proxy.service.extraPublicPorts }}
    {{- . | toYaml | trimSuffix "\n" | nindent 4 }}
    {{- end }}
```